Formula/r/radvd.rb

```diff
@@ -0,0 +1,74 @@
+class Radvd < Formula
+  desc "IPv6 Router Advertisement Daemon"
+  homepage "https://radvd.litech.org/"
+  url "https://github.com/radvd-project/radvd/releases/download/v2.20/radvd-2.20.tar.xz"
+  sha256 "25d2960fb977ac35c45a8d85b71db22ed8af325db7dbf4a562fb03eab2848dcd"
+  license "radvd"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+    strategy :github_latest
+  end
```

Comment:
We can omit default regex, or remove all and just check the github tag by default.
```suggestion
  livecheck do
    url :stable
    strategy :github_latest
  end
```